docs/01-overview/main-areas/observability/obsv-10-useful-links.md

```diff
@@ -4,7 +4,7 @@ title: Observability - Useful links
 
 If you're interested in learning more about the Observability area, check out these links:
 
-- Learn how to set up the [Monitoring Flow](../../../03-tutorials/00-observability) for your services in Kyma, in which you [observe your application metrics](../../../03-tutorials/00-observability/obsv-01-observe-application-metrics.md), [set up your Grafana Dashboard](../../../03-tutorials/00-observability/obsv-02-create-and-configure-grafana-dashboard.md), [define alerting rules](../../../03-tutorials/00-observability/obsv-03-define-alerting-rules-monitor.md), and eventually, [send notifications](../../../03-tutorials/00-observability/obsv-04-send-notifications.md). And remember to [clean up when you're done](../../../03-tutorials/00-observability/obsv-05-clean-up-configuration.md). 
+- Learn how to set up the [Monitoring Flow](../../../03-tutorials/00-observability.md) for your services in Kyma, in which you [observe your application metrics](../../../03-tutorials/00-observability/obsv-01-observe-application-metrics.md), [set up your Grafana Dashboard](../../../03-tutorials/00-observability/obsv-02-create-and-configure-grafana-dashboard.md), [define alerting rules](../../../03-tutorials/00-observability/obsv-03-define-alerting-rules-monitor.md), and eventually, [send notifications](../../../03-tutorials/00-observability/obsv-04-send-notifications.md). And remember to [clean up when you're done](../../../03-tutorials/00-observability/obsv-05-clean-up-configuration.md).
```

Comment:
All the links here (except for Monitoring Flow) return 404.

Reply:
Removed links to deleted documents.